src/main/java/com/j256/simplemagic/ContentType.java

```diff
@@ -953,5 +958,21 @@ public static ContentType fromFileExtension(String fileExtension) {
 		} else {
 			return type;
 		}
-	}
+        }
+        
+        /**
+         * Returns the references of the mime type.
```

Comment:
Can you flesh out this comment?  What is a "reference"?  Where do they come from?